L1Trigger/TrackerTFP/interface/LayerEncoding.h

```diff
@@ -0,0 +1,37 @@
+#ifndef L1Trigger_TrackerTFP_LayerEncoding_h
+#define L1Trigger_TrackerTFP_LayerEncoding_h
+
+#include "FWCore/Framework/interface/data_default_record_trait.h"
+#include "L1Trigger/TrackerTFP/interface/LayerEncodingRcd.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+
+#include <vector>
+
+namespace trackerTFP {
+
+  class LayerEncoding {
```

Comment:
Add top comment to this class, explaining what it does.

Reply:
done

Reply:
Top comment needs to give more detail.e.g.

"In bins of (tanL, z0) within each rapidity sector, this forms a list of which tracker layers (barrel=1-6, endcap=11-15)a particle is expected to cross, sorted in the order in which it would cross them. In any given bin, it can return the KF layer (= position 0-6 within this sorted order) of any given tracker layer.